src/Microsoft.Extensions.Logging.Console/ConsoleLoggerProvider.cs

```diff
@@ -52,7 +52,7 @@ private void OnConfigurationReload(object state)
             foreach (var logger in _loggers.Values)
             {
                 logger.Filter = GetFilter(logger.Name, _settings);
-                logger.IncludeScopes = _settings.IncludeScopes;
+                logger.IncludeScopes = _settings?.IncludeScopes ?? default(bool);
```

Comment:
Done, changed!

Just out of curiosity. Was the request made for coding style reasons, or are they any drawbacks for using `default(bool)` instead of `false`? (I suspect that theoretically `default(bool)` could change to `true`, but we both know that such breaking change would never be allowed).